articles/Introduction-to-time-series/index.md

```diff
@@ -0,0 +1,60 @@
+title: Introductions to time series
+description: This article will give users an introduction to what time series is in machine learning. It is fast rising deep learning concept with a broad scope of projects, and It will also give a detailed overview and basic implementations.
+
+### INTRODUCTION TO TIME SERIES
+
+![Simple](https://github.com/jamessandy/engineering-education/blob/new-article/articles/introduction-to-time-series/hero.jpg)
+
+In Machine learning, time series is a very crucial aspect but is often overlooked. They are useful in forecasting, stock market analysis even forex because it is the time component that makes it challenging to handle. This article will give a friendly introduction to time series.
+
+#### Tabel of Content
+1. Introductions to Time series
+2. Real-life applications of Time series
+3. Time-series Forecasting & Time-series Analysis
+4. Time Series Forecasting methods
+5. Components of Time Series
+6. Conclusion
+
+Time series can be defined as a sequence or series of data points that are ordered in time. In traditional machine learning, the dataset is a collection of observations, and predictions are made based on the new data, which likely will not be known until the future. The future is what is predicted, but all the previous observations are put into considerations. In time series, the dataset is different. Time-series adds a definite order dependence between observations.
+
+Time series are assumed to be generated at a spaced timing. When the data in time series are timed are regular, they are called **regular time series** and when they are not at a regular interval, they are called **irregular time series**
+
+#### Real-life applications of Time series
+1. Can be used to predicting the closing price of the stock at the end of the day
+2. Used in Predicting the number of product sales
```

Comment:
Avoid unnecessary capitalization, such as Predicting. If you want to capture the attention, use italics or bold.